src/integration_tests/integration_tests/integration_test_node.py

```diff
@@ -0,0 +1,662 @@
+import builtins
+import functools
+import importlib
+import os
+import signal
+import subprocess
+import sys
+import time
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Any, Optional, Tuple, Type, TypeVar, Union
+
+import rclpy
+import rclpy.node
+import yaml
+from rclpy.impl.rcutils_logger import RcutilsLogger
+from rclpy.node import Node
+
+from integration_tests.gen_ros_dtypes import gen_ros_dtypes
+
+# Generate and import datatypes
+gen_ros_dtypes()
+ROS_DTYPES_MOD = importlib.import_module("integration_tests.ros_dtypes")
```

Comment:
Why do you need to use importlib for this?

Reply:
* Need to import `gen_ros_dtypes()` and run it.
* Need to import `ros_dtypes`.
* The formatter and linter *really* don't like this if we're using normal `import` statements

Reply:
Could try importing in the function it is used? Or commenting `# noqa: EXX` to ignore the warning

Reply:
Fixed. The formatter made it really ugly the first time when I didn't have a comment above it.